### PR TITLE
Entity Readonly: Insert (new)

### PR DESCRIPTION
**Pull request by OlesiaKazmirchuk**
Create EntityReadonlyInsertTest class
Add Insert new readonly test

[User story](https://trello.com/c/gsQkZT09/16-entity-readonly-input-new-record)
[Test case](https://trello.com/c/NzEPsLoE/213-entity-readonly-input-new-record)
[Automatio test](https://trello.com/c/zkIt42IH/220-entity-readonly-input-new-record)

### CODE REVIEW

File: src/test/java/EntityReadonlyInsertTest.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        Assert.assertTrue(Boolean.parseBoolean(inpString.getAttribute("readonly")));
+        Assert.assertTrue(Boolean.parseBoolean(inpText.getAttribute("readonly")));
+        Assert.assertTrue(Boolean.parseBoolean(inpDec.getAttribute("readonly")));
+        Assert.assertTrue(Boolean.parseBoolean(inpInt.getAttribute("readonly")));
+        Assert.assertTrue(Boolean.parseBoolean(inpDate.getAttribute("readonly")));
+        Assert.assertTrue(Boolean.parseBoolean(inpDatetime.getAttribute("readonly")));
+
+        //TEST FAILS because button is enabled (which is not right)
+        //I want to do my first Pull request, that's why I comment code here:
+        //Assert.assertTrue(Boolean.parseBoolean(driver.findElement(By.xpath("//button[contains(text(),'+')]")).getAttribute("disabled")));
```
Comment (SergeiDemyanenko) [inline src/test/java/EntityReadonlyInsertTest.java:36]: Комментарии в коде не допустимы

**Review by SergeiDemyanenko — CHANGES_REQUESTED** [review]
Имя класса и имя метода теста необходимо привести в соответствие с правилами

File: src/test/java/EntityReadonlyInsertTest.java (hunk 1 above)
Comment (OlesiaKazmirchuk) [reply src/test/java/EntityReadonlyInsertTest.java]: I corrected everything. Could you please merge changes?

**Review by denisumanetssf — APPROVED** [review]
Test case is in Ignore because of the bug below

https://trello.com/c/VIdT25je/33-user-is-able-to-introduce-data-when-adds-new-readonly-record

**Pull request merged**